components/presenters/Docs/Masthead/MastheadMenuItem.tsx

```diff
@@ -18,24 +18,28 @@ export const MastheadMenuItem: React.FC<MastheadMenuItemProps> = ({
   const [showChildren, setShowChildren] = useState<boolean>(false)
 
   return (
-    <StyledMastheadMenuItem>
-      {React.cloneElement(link, {
-        passHref: true,
-        children: (
-          <StyledMastheadMenuLink>{link.props.children}</StyledMastheadMenuLink>
-        ),
-      })}
-      {children && (
-        <>
+    <StyledMastheadMenuItem $hasChildren={!!children}>
+      <div>
+        {React.cloneElement(link, {
+          passHref: true,
+          children: (
+            <StyledMastheadMenuLink>
+              {link.props.children}
+            </StyledMastheadMenuLink>
+          ),
+        })}
+        {children && (
           <StyledExpandButton
-            onClick={(_) => setShowChildren(!showChildren)}
+            onClick={(_: React.MouseEvent<HTMLButtonElement>) =>
+              setShowChildren(!showChildren)
+            }
             data-testid="masthead-menu-expand-button"
           >
             {!showChildren ? <IconExpandMore /> : <IconExpandLess />}
           </StyledExpandButton>
-          {showChildren && children}
-        </>
-      )}
+        )}
+      </div>
+      {children && showChildren && children}
```

Comment:
Although minor I wonder if abstracting `hasChildren` and doing `hasChildren && showChildren && children` is more readable.